src/components/Breadcrumbs/index.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import { Home } from 'react-feather';
 import { Link } from 'react-router-dom';
-import { SubjectType, GradeType, ClaimType } from './BreadcrumbModel';
 import { BreadcrumbLink } from './BreadcrumbLink';
+import { Styles, Colors } from '../../constants';
 
 /**
  * Properties for Breadcrumbs
  * @export
  * @interface BreadcrumbsProps
  */
 export interface BreadcrumbsProps {
-  subject?: SubjectType;
-  grade?: GradeType;
-  claim?: ClaimType;
+  subject?: string;
+  grade?: string;
+  claim?: string;
   target?: string;
 }
 
```

Comment:
For line 19, we should move this style object into the CSS for this component. You could give the `<Link/>` a classname, but I would probably just add a css selector for `<a/>` tags returned by the `<Link/>` component

Reply:
This has been updated